src/setupTests.js

```diff
@@ -3,3 +3,6 @@ import Adapter from 'enzyme-adapter-react-16'
 import 'jest-enzyme'
 
 configure({ adapter: new Adapter() })
+
+// prevent console.error output during test runs
+console.error = () => {}
```

Comment:
This can be dangerous, especially if there are errors but the test doesn't fail (like it can happen with react components). Better to do this with an env flag set when building/deploying.